_infra/helm/locust/locustfiles/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+[packages]
+google-cloud-storage = "*"
+locust = "*"
+bs4 = "*"
```

Comment:
This is installed in the locust container by the `pip_packages` definition, but for a standalone local run time we needed to add it to our Pipeline